wifiphisher/common/interfaces.py

```diff
@@ -465,6 +506,18 @@ def set_interface_mode(self, interface_name, mode):
         pyw.modeset(card, mode)
         pyw.up(card)
 
+    def add_card(self, interface):
```

Comment:
> it would be like so:
```python
 def is_interface_valid(self, interface_name, mode=None)
    ...
    self._active.add(interface_name)
```

I need to separate the internet card from the ap and deauth adapters since I will check if the card is in `unmanaged` state in the `NetworkManager.conf` for the deauth and ap adapters. If the internet card is also in the `_active` I'll raise the unnecessary exception for it.
Otherwise, I need to add one more property in `NetworkAdapter` to specify it is internet card, and don't check the `unmanaged` property for it.

> And maybe we should do a check to see if interface is connected to internet?

Yes, I have thought it before. Imagine a scenario, if the user use a card that connected to internet, he/she misuse this card as deauth/ap interface, should we raise exception after it choose as deauth/ap interface?

Also user may not use `NetworkManager` to connect to internet, he/she may use `wpa_supplicant` + `dhclient` then I cannot determine by `NetowrkManager dbus` to determine the card is on internet or not.